src/Resources/public/core.js

```diff
@@ -941,20 +941,24 @@ var Backend =
 				alert('Could not find the SimpleModal frame');
 				return;
 			}
-			ul = frm.document.getElementById(opt.id);
+			ul = frm.document.getElementById('tl_select');
```

Comment:
I think it’s very likely that somebody copied the `tl_content::pagePicker()` method to use it for an extension or in a project, this would include `\'id\':\'' . $dc->field . '\',` too. Not supporting `opt.id` is be a BC break I think.

How about using `tl_select` as a fallback? 
`frm.document.getElementById(opt.id || 'tl_select')`